flows/MDS/bird.py

```diff
@@ -29,9 +29,8 @@
 current_environment = os.getenv("PREFECT_CURRENT_ENVIRONMENT", "staging")
 
 # Set up slack fail handler
-handler = slack_notifier(only_states=[Failed])
+handler = slack_notifier(only_states=[Failed, TriggerFailed])
```

Comment:
I would like to try to see if this state happens when the flows don't start running at all.

Reply:
[It looks like](https://docs.prefect.io/api/latest/engine/state.html#state-2) `TriggerFailed` is a substate of the `Failed` state. Possible there's a bug and this isn't working how they thought?
```
my_state = TriggerFailed()
my_state.is_failed() # returns True
```

The silent error in the task from last night is actually technically in a `pending` state?
<img width="259" alt="Screen Shot 2022-02-18 at 3 43 22 PM" src="https://user-images.githubusercontent.com/30810522/154765295-8066265d-51e6-4730-a46e-238f0b20bcc9.png">